generator/vulkan/render.zig

```diff
@@ -1035,7 +1020,7 @@ fn Renderer(comptime WriterType: type) type {
                 \\    var self: Self = undefined;
                 \\    inline for (std.meta.fields(Dispatch)) |field| {{
                 \\        const name = @ptrCast([*:0]const u8, field.name ++ "\x00");
-                \\        const cmd_ptr = loader({s}name) orelse return error.InvalidLoader;
+                \\        const cmd_ptr = loader({s}name) orelse return error.CommandLoadFailure;
```

Comment:
I think this name makes more sense than `InvalidCommand`, as whenever I had this issue my problem was actually with a faulty loader function - so this error includes that possibility.